src/libYARP_dev/src/devices/ControlBoardWrapper/ControlBoardWrapper.cpp

```diff
@@ -3320,13 +3321,14 @@ bool ControlBoardWrapper::disableAmp(int j)
         return false;
 
     // Use the newer interface if available, otherwise fallback on the old one.
-    if(p->iMode2) {
+    if(p->iMode2)
         return p->iMode2->setControlMode(off+p->base, VOCAB_CM_IDLE);
-    }
-    if (p->pos) {
-        return p->amp->disableAmp(off+p->base);
-    }
-    return false;
+    else
+        if (p->pos)
+        {
+            return p->amp->disableAmp(off+p->base);
+        }
+        return false;
```

Comment:
I am confused. `return false` is outside the scope of the `else`, while the identation suggest that is inside the scope. Always adding brackets following `if` and `else`'s simplifies reading this kind of code.